R/pkg/R/SQLContext.R

```diff
@@ -328,6 +328,7 @@ setMethod("toDF", signature(x = "RDD"),
 #' It goes through the entire dataset once to determine the schema.
 #'
 #' @param path Path of file to read. A vector of multiple paths is allowed.
+#' @param ... additional external data source specific named properties.
```

Comment:
Yes, it was not. I am not very sure on this (as actually I am not used to this CRAN check). My guess is, it seems they combine the arguments? For example, Parquet API is as below:

```r
 #' @param path path of file to read. A vector of multiple paths is allowed.
 #' @return SparkDataFrame
 #' @rdname read.parquet
...
read.parquet.default <- function(path, ...) {
```

```r
 #' @param ... argument(s) passed to the method.
 #' @rdname read.parquet
...
 parquetFile.default <- function(...) {
```

It complained about duplicated `@params` when I tried to add `@param ...` to `read.parquet.default`. So, I ended up with removing this back.

On the other hand, for JSON APIs,

```r
 #' @rdname read.json
 #' @name jsonFile
...
 jsonFile.default <- function(path) {
```

```r
#' @param path Path of file to read. A vector of multiple paths is allowed.
#' @param ... additional external data source specific named properties.
#' @return SparkDataFrame
#' @rdname read.json
...
read.json.default <- function(path, ...) {
```

It seems `jsonFile` does not describe `@param`. So, I think it passed.

If you meant another problem, could you please guide me?